File: HackerHank/PlusMinus/SolutionPlusMinus.cpp
#include <vector>
#include <iostream>

using namespace std;

void plusMinus(vector<int> arr)
{
    float neg = 0, zeros = 0, pos = 0, total = 0;
    for (int i = 0; i < arr.size(); i++)
    {
        if (arr[i] > 0)
            pos++;
        else if (arr[i] < 0)
            neg++;

        else
            zeros++;
        total++;
    }
    cout << pos / total << endl;
    cout << neg / total << endl;
    cout << zeros / total << endl;
}

File: HackerHank/PlusMinus/SolutionPlusMinus.py
def plusMinus(arr):
    # Write your code here
    zeros = 0
    pos = 0
    neg = 0
    total = 0
    for i in range(0, len(arr)):
        if(arr[i] == 0):
            zeros += 1
        elif(arr[i] > 0):
            pos += 1
        elif(arr[i] < 0):
            neg += 1
        total += 1
    print(pos/total)
    print(neg/total)
    print(zeros/total)


myArr = [1, 2, 3, 4, 5, 6]
plusMinus(myArr)
